src/primitive_core.cc

```diff
@@ -1353,6 +1353,21 @@ PRIMITIVE(float_sqrt) {
   return Primitive::allocate_double((double) sqrt(receiver), process);
 }
 
+PRIMITIVE(float_ceil) {
+  ARGS(double, receiver);
+  return Primitive::allocate_double((double) ceil(receiver), process);
```

Comment:
Why do we need the C-style casts of the result here and in other places?

Reply:
done.